packages/eslint-plugin/test/rules/jsx-import.test.ts

```diff
@@ -231,13 +230,30 @@ let ele = <div css={{}} />
     },
     {
       code: `
+/** @jsx jsx */
+import * as emotion from '@emotion/react'
+let ele = <div css={\`color:hotpink;\`} />
+      `.trim(),
+      errors: [
+        {
+          messageId: 'cssPropWithPragma'
+        }
+      ],
+      output: `
+/** @jsx jsx */
+/** @jsx emotion.jsx */
```

Comment:
This one seems like a new test case. Was this a bug and it got fixed here? From what I understand the issue could be about the namespace import being used in a file with an invalid pragma for such an import. I've skimmed through the related code and I don't see any particular changes that would fix this - although I'm not super familiar with this code, so I could have missed the fix.

Reply:
This should have been for either an actual bug or to ensure coverage, but I can't get it to fail so happy to remove it if you like